osu.Game/Graphics/UserInterface/BackButton.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) 2007-2016 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu-framework/master/LICENCE
+
+using OpenTK;
+using OpenTK.Graphics;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Transformations;
+using osu.Framework.Input;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    // Basic back button as it was on stable (kinda). No skinning possible for now
+    class BackButton : ClickableContainer
+    {
+        private TextAwesome icon;
+
+        private Container leftContainer;
+        private Container rightContainer;
+
+        public Vector2 ExtendLength;
+        public Vector2 InitialExtendLength;
+
+        private Color4 colorBright = new Color4(238, 51, 153, 255);
+        private Color4 colorDark = new Color4(195, 40, 140, 255);
+        private const double transformTime = 300.0;
+        private const int pulseLength = 250; // FIXME: remove when bpm-based pulsing will be possible

```

Comment:
pulse_length. also no need for FIXME; there needs to be a default for cases where timing data is not available.